src/LaunchDarkly.Logging/LaunchDarkly.Logging.csproj

```diff
@@ -7,6 +7,9 @@
     <OutputType>Library</OutputType>
     <PackageId>LaunchDarkly.Logging</PackageId>
     <Description>LaunchDarkly logging abstraction</Description>
+    <Company>LaunchDarkly</Company>
+    <Authors>LaunchDarkly</Authors>
+    <Owners>LaunchDarkly</Owners>
```

Comment:
This looks redundant, but omitting Authors and Owners causes some NuGet package browsers to use the package name as the default, which looks weird (e.g. "LaunchDarkly.Logging by LaunchDarkly.Logging").